.github/workflows/validate-pr.yml

```diff
@@ -0,0 +1,56 @@
+# Validates the format of a PR title and body. Because the title will become the default commit message on squash-merge,
+# it should follow the conventional commit format and be 70 characters long or fewer. Because the body will later be
+# used in creating docs-site release notes, it must have a summary paragraph (70+ characters) followed by `\n---`.
+
+name: "Validate PR"
+on:
+  pull_request_target:
```

Comment:
Should this be `pull_request`

Reply:
Yes, agreed. It doesn't need more than read-only permissions on the PR regardless of whether the PR is coming from a fork. (E.g. it's not removing labels or adding comments.) Might not hurt to look at our other cases of pull_request_target with the least-privilege principle in mind as well.